src/background.js

```diff
@@ -351,6 +351,10 @@ async function areAllStringsTranslated () {
 }
 
 async function updateBrowserActionIcon (tab) {
+  if (tab.incognito) {
+    browser.browserAction.disable();
```

Comment:
We need a corresponding `browserAction.enable()` call or else the panel gets stuck in `disable`d state after the user closes their PBM window.

Reply:
Whoops. I've added the `tab.id` to the disable call, so the browserAction is only disabled in incognito tabs and not globally.